venv/cmd_project/scripts/command_line.py

```diff
@@ -0,0 +1,39 @@
+from tokenize import group
+import click
+import csv
+from sorting import MergeSort
+def csv_read_details(file_name):
+    data = []
+    with open(file_name, 'r') as file:
+                reader = csv.reader(file)
+                header = next(reader)
+                for row in reader:
+                    data.append(row)
+    sorted = MergeSort(data)
```

Comment:
Separate this out. Have the function just do a simple thing. The read or write functions should not depend on the sort